vcd/datasource_vcd_catalog.go

```diff
@@ -40,6 +41,21 @@ func datasourceVcdCatalog() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"publish_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "True enables external publication",
```

Comment:
Would be nice if you could add more explanation of publication to where (same ask for the same line in resource down below).

Reply:
It's name and explanation from VCD, not mine. Yet overall means that this catalog will be accessible for subscribe from different VCD.

Reply:
Once you extend the description in `catalog.html.markdown`, would help someone if you paste it here as well.

Reply:
I am not keen on explaining functionality I am not expert, but will try to take something usable from docs.

Reply:
changed